staging/src/k8s.io/apiserver/pkg/util/flowcontrol/request/mutating_work_estimator.go

```diff
@@ -129,3 +138,12 @@ func (e *mutatingWorkEstimator) estimate(r *http.Request, flowSchemaName, priori
 		AdditionalLatency: additionalLatency,
 	}
 }
+
+func isRequestExemptFromEvents(requestInfo *apirequest.RequestInfo) bool {
```

Comment:
event seems more generic, like audit event, does `isRequestExemptFromWatchEvents` sound better?

Reply:
+1 for being more explicit